packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -82,15 +42,15 @@ void main() {
           child: CustomScrollView(
             slivers: <Widget>[
               CupertinoSliverRefreshControl(
-                builder: builder,
+                builder: mockHelper.builder,
               ),
               buildAListOfStuff(),
             ],
           ),
         ),
       );
 
-      verifyNoMoreInteractions(mockHelper);
+      mockHelper.finishInteractions = true;
```

Comment:
nit: maybe call this throwOnInteractions?

Reply:
actually, this is kinda strange, isn't the intention of `verifyNoMoreInteractions` to insure that only expected invocations have happened so far? It's not really making any statements about the future, right?

Could we better express this as something like `expect(mockHelper.invocations, hasLength(<whatever>))`?

Reply:
Same elsewhere.

Reply:
Ahh right, I got this mixed up with something else. In that case, yeah checking the length would be the right way to do it